pyfolio/perf_attrib.py

```diff
@@ -172,7 +177,9 @@ def show_perf_attrib_stats(returns, positions, factor_returns,
     perf_attrib_stats, risk_exposure_stats =\
         create_perf_attrib_stats(perf_attrib_data, risk_exposures)
 
-    print_table(perf_attrib_stats)
+    print_table(perf_attrib_stats[:perf_attrib_stats.index.get_loc(
+        "Cumulative Specific Return"
```

Comment:
Can we write this using `drop`?
```
perf_attrib_stats.drop(['Cumulative Specific Return', 'Cumulative Common Return', 'Total Returns'])
```
Should be clearer, as well as not dependent on the order of the rows.

Reply:
👍  fixed